### PR TITLE
Move awerb question to admin ppl endorsement

### PR DESCRIPTION
**Pull request by wheelsandcogs**
waiting on 
~https://github.com/UKHomeOffice/asl-public-api/pull/272~
~https://github.com/UKHomeOffice/asl-workflow/pull/301~

### CODE REVIEW

File: pages/task/schema/confirm.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
     return nextStep && nextStep.commentRequired;
   };
 
-  return {
+  const awerbRequired = task => {
+    // awerb question is required for project applications / amendments that haven't yet been marked as awerbed
+    const model = get(task, 'data.model');
+    const status = get(task, 'status');
+    const wasAwerbed = get(task, 'data.meta.awerb') === 'Yes';
+    return model === 'project' && status === 'awaiting-endorsement' && !wasAwerbed;
```
Comment (lennym) [inline pages/task/schema/confirm.js:15]: Why might a project task require endorsement but already have been awerbed?

File: pages/task/schema/confirm.js [hunk 2]
```diff
@@ -14,4 +25,18 @@ module.exports = (task, chosenStatus) => {
...
     }
   };
+
+  const isAmendment = get(task, 'type') === 'amendment';
+
+  if (awerbRequired(task)) {
+    schema = {
+      awerb: {
+        ...getAwerbQuestion(isAmendment),
+        automapReveals: true
```
Comment (lennym) [inline pages/task/schema/confirm.js:35]: Is there a reason we want this here, but not in the other instances of the same thing?

Comment (wheelsandcogs) [reply pages/task/schema/confirm.js:35]: This is a new feature @joefitter added that means you don't need to manually map the reveals, e.g. https://github.com/UKHomeOffice/asl-pages/blob/master/pages/project-version/update/submit/views/index.jsx#L6-L26

Comment (lennym) [reply pages/task/schema/confirm.js:35]: How come it is _only_ used here and not in the other places we ask the question though?

File: pages/task/schema/confirm.js (hunk 1 above)
Comment (wheelsandcogs) [reply pages/task/schema/confirm.js:15]: This schema is shared by all the task confirm pages, and we only want to show the Awerb question when a project is being endorsed.

**Pull request merged**